plugins/processors/awsapplicationsignals/internal/resolver/kubernetes.go

```diff
@@ -523,38 +481,48 @@ func getKubernetesResolver(platformCode, clusterName string, logger *zap.Logger)
 		// jitter calls to the kubernetes api
 		jitterSleep(jitterKubernetesAPISeconds)
 
-		sharedInformerFactory := informers.NewSharedInformerFactory(clientset, 0)
+		sharedInformerFactory := informers.NewSharedInformerFactoryWithOptions(clientset, 0, informers.WithTweakListOptions(func(options *metav1.ListOptions) {
+			options.ResourceVersion = ""
+		}))
 		podInformer := sharedInformerFactory.Core().V1().Pods().Informer()
+		err = podInformer.SetTransform(filterPodIPFields)
+		if err != nil {
+			logger.Error("failed to set custom transform into pod store", zap.Error(err))
+		}
```

Comment:
will Informer fallback to the default caching behavior with the full Pod metadata stored if `SetTransform` failed?

Reply:
Yes.